ios/FluentUI/People Picker/AvatarViewVnext.swift

```diff
@@ -0,0 +1,340 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc(AvatarVnextStyle)
+/// Pre-defined styles of the avatar
+public enum AvatarVnextStyle: Int, CaseIterable {
+    case `default`
+    case noring
+    case unauthenticated
+    case overflow
+    case group
+}
+
+@objc(MSFAvatarVnextSize)
+/// Pre-defined sizes of the avatar
+public enum AvatarVnextSize: Int, CaseIterable {
+    case xsmall
+    case small
+    case medium
+    case large
+    case xlarge
+    case xxlarge
+}
+
+@objc(MSFAvatarVnextPresence)
+public enum AvatarVnextPresence: Int, CaseIterable {
+    case none
+    case available
+    case away
+    case busy
+    case doNotDisturb
+    case outOfOffice
+    case offline
+    case unknown
+    case blocked
```

Comment:
What's the reason or scenario for making it a separate boolean instead of just another presence enum?
What would it change in the way the Avatar looks or behaves?

Reply:
Just started a follow up conversation with the design team on whether we should support "Out Of Office" variations for each of the presence options.

Reply:
Hey @Saadnajmi. I have created #339 to track this issue.